resolwe/flow/managers/listener/listener.py

```diff
@@ -560,11 +559,12 @@ def __init__(
         if zmq_socket is None:
             zmq_context: zmq.asyncio.Context = zmq.asyncio.Context.instance()
             # Start the authenticator.
-            auth = AsyncioAuthenticator(zmq_context)
-            auth.start()
-            auth.configure_curve_callback(
-                domain="*", credentials_provider=CurveCallback()
-            )
+            if ZMQAuthenticator.has_instance() is None:
```

Comment:
Why do you check whether the class was initialized? It defeats the purpose of making the class a singleton.

Reply:
This was added since tests were failing due to the env bug. Removed.